lib/ansible/module_utils/facts.py

```diff
@@ -604,6 +604,10 @@ class Distribution(object):
     This is unit tested. Please extend the tests to cover all distributions if you have them available.
     """
 
+    # every distribution name mentioned here, must have one of
+    #  - allowempty == True
+    #  - be listed in SEARCH_STRING
+    #  - have a function get_distribution_DISTNAME implemented
```

Comment:
Thanks, do you mean a test case for arch linux or do you mean running the existing test suite on arch linux and perhaps more distros would cover this ?